R/zzz.R

```diff
@@ -1,5 +1,27 @@
 .lockbox_env <- new.env()
 
+.get_safely_from_envir <- function(target, default = NULL) {
+  res <- tryCatch(
+    get(target, envir = .lockbox_env)
+  , error = function(e) {default})
+}
+
+set_session_id <- function() {
+  if (is.null(.get_safely_from_envir("session_id"))) {
+    if (requireNamespace("uuid", quietly = TRUE)) {
+      assign("session_id", uuid::UUIDgenerate(), envir = .lockbox_env)
```

Comment:
Use of the [`%||%` infix](https://github.com/robertzk/lockbox/blob/master/R/utils.R#L1) should cover that, right? It checks implicitly for whether the left-hand side is null, and if it is, only then proceeds to use the right hand side.